files-community/Files
# Code Quality: Introduced AppThemeModeService

## Summary

This is one of the efforts to migrate static Helpers to singleton Services.

## Task Checklist

- [x] Introduced `AppThemeModeService` : `IAppThemeModeService`
- [x] Removed SettingsViewModel, which had theme changed event handler and has been marked as obslete.

## Steps To Validate Changes

1. Launch the app
2. See the sections you have expanded are expanded expectedly
3. Go to Settings dialog
4. Change theme mode
5. See theme will be changed both in main app window and dialog

## Known Issues

_N/A_

## PR Checklist

- [x] **Development**: Related: #13490

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
